pkg/extensions/extension_scrub.go

```diff
@@ -20,7 +20,7 @@ func EnableScrubExtension(config *config.Config, log log.Logger, storeController
 	sch *scheduler.Scheduler,
 ) {
 	if config.Extensions.Scrub != nil &&
-		config.Extensions.Scrub.Interval != 0 {
+		*config.Extensions.Scrub.Enable {
```

Comment:
Also add the Interval check, what if it was set to zero by mistake?

Reply:
We have a minimum value for this interval which is 2h, so if this interval is set to zero in config then it will be automatically replaced by this minimum value.